sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -548,6 +551,57 @@ def receive_files(self, src, dst, delete_dst=False,
 
     # pylint: disable=too-many-arguments,unused-argument
     def send_files(self, src, dst, delete_dst=False, preserve_symlinks=False, verbose=False):
-        # TODO: make this method create configmap with provided file and then
-        #       add it to the template and mount it's files.
-        raise NotImplementedError()
+        """Mount single files to a 'dynamic'aly created pods.
+
+        'src' and 'dst' params must contain filename.
+        'delete_dst', 'preserve_symlinks' and 'verbose' are ignored.
+
+        Usage example:
+
+            loader.remoter.send_files(
+                src='/home/sct-runner/scylla-cluster-tests/data_dir/c-s_lwt_basic.yaml',
+                dst='/tmp/c-s_lwt_basic.yaml',
+            )
+        """
+        # Generate name for the configMap
+        cm_name = f"cm--{self.pod_name_template}--{generate_random_string(5).lower()}"
+        filename = dst.split('/')[-1]
+
+        # Create configMap from the 'src' file
+        self.kluster.kubectl(
```

Comment:
we are running kubectl local inside SCT docker, right ? (i.e. not like helm that runs on it's own docker instance)

Reply:
It runs on the host machine directly using `kubectl` binary directly